src/drivers/uwb/uwb_sr150/module.yaml

```diff
@@ -32,22 +31,60 @@ parameters:
 
         UWB_INIT_OFF_Z:
           description:
-            short: UWB sensor Y offset in body frame
+            short: UWB sensor Z offset in body frame
             long: UWB sensor positioning in relation to Drone in NED. Z offset.
           type: float
           unit: m
           decimal: 2
           increment: 0.01
           default: 0.00
 
-        UWB_INIT_OFF_YAW:
+        UWB_SENS_ROT:
           description:
-            short: UWB sensor YAW offset in body frame
-            long: UWB sensor positioning in relation to Drone in NED. Yaw rotation in relation to direction of FMU.
-          type: float
-          unit: deg
-          decimal: 1
-          increment: 0.1
-          default: 0.00
-
-
+            short: UWB sensor orientation
+            long: The orientation of the sensor relative to the forward direction of the body frame. Look up table in src/lib/conversion/rotation.h
+                  Default position is the antannaes downward facing, UWB board parallel with body frame.
+          type: enum
+          values:
+            0: ROTATION_NONE
```

Comment:
@bresch is there a way to re-use the rotation value definitions that already exist for other sensors?

Reply:
Could we get an answer here?